src/sync/list.rs

```diff
@@ -1,125 +1,175 @@
 use std::sync::atomic::Ordering::{Acquire, Relaxed, Release};
 
-use {Atomic, Owned, Ptr, Scope};
+use {Atomic, Owned, Ptr, Scope, unprotected};
+use util::cache_padded::CachePadded;
+
 
 /// An entry in the linked list.
-pub struct Entry<T> {
+struct Node_<T> {
```

Comment:
I `CachePadded` this `Node_` to make `Node`.  I also don't like this name, but couldn't find a better one.  Would you please suggest one?